tests/load/pipeline/test_merge_disposition.py

```diff
@@ -263,8 +272,16 @@ def _get_shuffled_events(shuffle: bool = dlt.secrets.value):
         yield issues
 
 
-@pytest.mark.parametrize('destination_name,github_resource', itertools.product(ALL_DESTINATIONS, [github_repo_events, github_repo_events_table_meta]))
-def test_merge_with_dispatch_and_incremental(destination_name: str, github_resource: DltResource) -> None:
+
+@pytest.mark.parametrize("destination_config", destinations_configs(default_configs=True), ids=lambda x: x.name)
+@pytest.mark.parametrize("github_resource",[github_repo_events, github_repo_events_table_meta])
+def test_merge_with_dispatch_and_incremental(destination_config: DestinationTestConfiguration, github_resource: DltResource) -> None:
+
+    # for athena we need to skip this test, as athena does not want a query string longer than 262144 (bytes?) and this is the
```

Comment:
what is failing here? getting table counts? btw. we could set correctly this:
caps.max_query_length = 16 * 1024 * 1024
caps.is_max_query_length_in_bytes = True

also this is merge resource. this means that for complicated datasets the merge resource will not work (or we'll be executing it line by line anyway, right)?

Reply:
'INSERT INTO "github_3_202308141258537531"."_dlt_version"(version_hash, schema_name, version, engine_version, inserted_at, schema) is failing, because the serialised schema string is too long. So I would say this can not be solved, since this is a single command that is too long.

Reply:
SqlJobClient will compress the schema string if it goes beyond `max_text_data_type_length` so setting it to correct value will help :) anyway I'll try to do that and then we'll see